Leanplum/Cache/LPFileManager.m

```diff
@@ -1,41 +1,10 @@
-//
-//  LPFileManager.m
-//  Leanplum
-//
-//  Created by Andrew First on 1/9/13.
-//  Copyright (c) 2013 Leanplum, Inc. All rights reserved.
-//
-//  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
i _think_ we need these licenses here for the open source files

Reply:
The reason its not needed, is because the file is changed a lot.